services/web/src/pages/registro/tutor.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { PublicLayout } from "components/layouts/PublicLayout";
+import { RegisterPageContent } from "components/contents/RegisterPageContent";
+import { StringSchema } from "yup";
+
+interface roleNameProps {
+  roleName: string,
+}
+
+export default function RegisterPage() {
+  return (
+    <PublicLayout>
+      <RegisterPageContent roleName="TUTOR"/>
```

Comment:
```suggestion
      <RegisterPageContent roleName={UserRoleName.TUTOR}/>
```
Mejor asi, por si se llega cambiar el enum por alguna razon no truene estos registros